src/test/unit/api/accounts/utils.js

```diff
@@ -0,0 +1,65 @@
+const request = require('request-promise-native');
+const OpenAPIResponseValidator = require('openapi-response-validator').default;
+
+const postAccountsBody = require('./data/postAccountsBody');
+
+/**
+ *
+ * @param reqInbound
+ * @param reqOutbound
+ * @param apiSpecsOutbound
+ * @return {function(...[*]=)}
+ */
+const createPostAccountsTester = ({reqInbound, reqOutbound, apiSpecsOutbound}) =>
+    /**
```

Comment:
@kirgene I think I would prefer to see the { } style with an explicit "return"...does this even work without?